src/wincurse.cpp

```diff
@@ -251,6 +284,26 @@ LRESULT CALLBACK ProcessMessages(HWND__ *hWnd,unsigned int Msg,
                 lastchar = code;
         }
         return 0;
+        
+    case WM_SIZE:
+    case WM_SIZING:
+        RECT WndRect;
+        if( GetClientRect( WindowHandle, &WndRect ) ) {
+            TERMINAL_WIDTH = WndRect.right / fontwidth;
+            TERMINAL_HEIGHT = WndRect.bottom / fontheight;
+            WindowWidth = TERMINAL_WIDTH * fontwidth;
+            WindowHeight = TERMINAL_HEIGHT * fontheight;
+            catacurses::resizeterm();
+            create_backbuffer();
+            SetBkMode(backbuffer, TRANSPARENT);//Transparent font backgrounds
+            SelectObject(backbuffer, font);//Load our font into the DC
+            color_loader<RGBQUAD>().load( windowsPalette );
+            if( SetDIBColorTable(backbuffer, 0, windowsPalette.size(), windowsPalette.data() ) == 0 ) {
+                throw std::runtime_error( "SetDIBColorTable failed" );
```

Comment:
This is throwing from a callback function that is called from non-C++ code. And the exception will leave the C++ code and that basically invokes [Undefined Behaviour](https://stackoverflow.com/questions/10956062).

Reply:
I was winging it with wincurse.cpp, honestly. Reading M$ Windows API references is like sticking a red hot poker in my eyes.